lib/cfnvpn/templates/lambdas/lib/slack.py

```diff
@@ -35,8 +37,12 @@ def post_event(self, message, state, error=None):
             colour = '#3ead3e'
 
         text = f'Message: {message}\nState: {state}'
+        
         if error:
             text += f'\nError: {error}'
+        
+        if support_case:
+            text += f'\nSupport Case: <https://console.aws.amazon.com/support/cases#/{support_case}|{support_case}>'
```

Comment:
Is this correct? "/{support_case}|{support_case}"

Reply:
it's slack's format for generating links `<link|text>`